tests/test_statcast_pitching_spin.py

```diff
@@ -0,0 +1,126 @@
+import pybaseball.statcast_pitcher_spin as spin
+import unittest
+import pandas as pd
+import numpy as np
+import logging
+
+SIG_DIG = 4 # Required because of read_csv bug in pandas
+
+rounding_error_columns = ['vxR', 'vyR', 'vxbar', 'vybar', 'vbar']
+
+class TestStatcastPitchingSpinCalcs(unittest.TestCase):
+
+	# Helper Methods
+	def compare_columns(self, df1, df2, name):
+		return df1[name].equals(df2[name])
+
+	def compare_almost_equal(self, df1, df2, name):
+		"""
+			Almost equal was necessary due to rounding errors that would result
+			from one value being calculated as .499999 and the next as .500000
+
+			Printing `comp_df.query('diff > 0.0000`) will show that only 1 or 2
+			values per calcuation rely on this function; the others are exactly
+			equal
+		"""
+
+		comp_df = pd.DataFrame()
+		comp_df['left'] = df1[name].round(SIG_DIG)
+		comp_df['right'] = df2[name].round(SIG_DIG)
+		comp_df['diff'] = comp_df['left'] - comp_df['right']
+		comp_df['diff'] = comp_df['diff'].abs().round(SIG_DIG)
+		# print(comp_df.query('diff > 0.0000'))
+		return comp_df.query('diff > .0001').empty
```

Comment:
It would probably be simpler to use pandas `assert_frame_equal` with the `check_exact` and `atol` parameters:

https://pandas.pydata.org/pandas-docs/stable/reference/api/pandas.testing.assert_frame_equal.html